src/Sidebar.js

```diff
@@ -72,7 +63,13 @@ function Sidebar() {
         <hr className="break-line" ></hr>
         <SidebarOptions Icon={AddIcon} title="Add Channels" />
         {channels.map((channel) => {
-          <SidebarOptions title={channel.name} id={channel.id} />
+          return (
```

Comment:
This was bug 3. You were not returning the item. So nothing was visible even when the first bug was solved. This took me 5 mins to see. Is subtle but is a bug : )

Reply:
I clearly need to do more practice on that. I will keep in my mind not to repeat it.